app/src/main/java/com/mk/tasky/agenda/data/mapper/AttendeeMapper.kt

```diff
@@ -0,0 +1,29 @@
+package com.mk.tasky.agenda.data.mapper
+
+import com.mk.tasky.agenda.data.local.entity.AttendeeEntity
+import com.mk.tasky.agenda.domain.model.Attendee
+import com.mk.tasky.agenda.util.TimeUtil
+
+// TODO: Event ID should be removed from here and be added to the attendee as soon as they get added on the event
+fun Attendee.toEntity(eventId: String): AttendeeEntity {
```

Comment:
Why does `Attendee` not have an `eventId` parameter? As soon as you add one, you're on the event detail page and you know which event they belong to 🤔

Reply:
Removed eventId since one attendee can have many events